resources/xip-patch/templates/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: xip-patch
+rules:
+- apiGroups: ["*"]
```

Comment:
Specify api groups here. With `*` this script would have access to all resources named `configmaps` and `scripts`, regardless of their apiGroup. In Knative for exmaple is resource `service` and we don't want the script to have access to it.